<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="QB-CLT-1378" failures="1" tests="2" name="Login to the application with valid credentials sauce50" time="26.17" errors="0" timestamp="2023-03-21T13:35:02 IST">
  <testcase classname="TestCase" name="Login_to_the_application_with_invalid_credentials_sauce51" time="11.467"/>
  <testcase classname="TestCase" name="Login_to_the_application_with_valid_credentials_sauce50" time="10.479">
    <failure type="java.lang.AssertionError" message="expected [Productss] but found [Products]">
      <![CDATA[java.lang.AssertionError: expected [Productss] but found [Products]
at TestCase.Login_to_the_application_with_valid_credentials_sauce50(TestCase.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- Login_to_the_application_with_valid_credentials_sauce50 -->
</testsuite> <!-- Login to the application with valid credentials sauce50 -->
